README.rst

```diff
@@ -250,6 +250,16 @@ Weekly Developer meeting
 * Weekly, Wednesday, 8:00 am PT, 11:00 am ET, 5:00 pm CEST
 * `Join zoom <https://zoom.us/j/596609673>`_
 
+Governance
+----------
+The Cilium project is governed by a group of `maintainers and commiters <https://github.com/cilium/cilium/blob/master/MAINTAINERS.md.>`_.
+How they are selected and govern is outlined in our `governance document <https://github.com/cilium/cilium/blob/master/Documentation/contributing/governance/commit_access.rst>`_.
+
+Adopters
+--------
+Adopters of the Cilium project who are deploying it in production and their use cases
+can be found in `Users <https://github.com/cilium/cilium/blob/master/USERS.md>`_.
```

Comment:
The list is not exhaustive.
```suggestion
A list of adopters of the Cilium project who are deploying it in production, and of their use cases,
can be found in `Users <https://raw.githubusercontent.com/cilium/cilium/master/USERS.md>`__.
```
(Again, raw file or GitHub-rendered, I don't mind much.)